recipe/meta.yaml

```
@@ -14,17 +14,17 @@ build:
 
 requirements:
   run:
-    - cuda-cudart-static 12.1.105
-    - cuda-nvrtc-static 12.1.105
-    - libcublas-static 12.1.3.1   # [linux]
-    - libcufft-static 11.0.2.54     # [linux]
-    - libcufile-static 1.6.1.9     # [linux64]
-    - libcurand-static 10.3.2.106    # [linux]
-    - libcusolver-static 11.4.5.107  # [linux]
-    - libcusparse-static 12.1.0.106  # [linux]
-    - libnpp-static 12.1.0.40       # [linux]
-    - libnvjitlink-static 12.1.105
-    - libnvjpeg-static 12.2.0.2    # [linux]
+    - cuda-cudart-static 12.2.140
+    - cuda-nvrtc-static 12.2.140
+    - libcublas-static 12.2.5.6   # [linux]
+    - libcufft-static 11.0.8.103     # [linux]
+    - libcufile-static 1.7.2.10     # [linux64]
```

Should this be updated to include `linux_aarch64`?

```suggestion
    - libcufile 1.7.2.10     # [linux64 or aarch64]
```